lib/api/apiUtils/bucket/bucketCreation.js

```diff
@@ -172,8 +183,6 @@ export function createBucket(authInfo, bucketName, headers,
 
     if (locationConstraint !== undefined) {
         bucket.setLocationConstraint(locationConstraint);
-    } else if (usEastBehavior) {
-        bucket.setLocationConstraint('us-east-1');
```

Comment:
I am double checking against AWS but I don't think amazon save the location to 'us-east-1' if user do not send any location.